### DISCUSSION TITLE
WASM bindings

### DISCUSSION THREAD

**Discussion by seanieb** [openmls/openmls#1175] [Q&A] [answered]
Hi Folks 👋

Has anyone tried to make wasm bindings for the library yet? If so are any tricks that would be helpful? 

Thank you!

**Comment by raphaelrobert** [comment]
The CI compiles for WASM, but there is no WASM support beyond that at this point. I think #487 is somewhat related.

**Comment by beltram** [comment, marked as answer]
We have made [this branch](https://github.com/wireapp/openmls/tree/feat/async) which works for `wasm32-unknown-unknown`. It required moving to async for keystore access and a few other nits. But it is now way behind master.
Hope this helps !
